charm/docs/index.md

```diff
@@ -1,64 +1,106 @@
-# Table of Contents
-1. [repo-policy-compliance](#introduction) 
+A [Kubernetes charm](https://canonical-juju.readthedocs-hosted.com/en/latest/user/reference/charm/charm-taxonomy/#kubernetes)
+that checks if a GitHub repository aligns with a chosen set of policies for workflow runs.
+
+When using the `github-runner` charm to deploy and manage your self-hosted runners in OpenStack mode,
+the self-hosted runners can execute arbitrary code. This may hurt compliance. Deploying the
+`repo-policy-compliance`charm, configuring it for OpenStack mode and integrating it with the
+`github-runner` charm ensures that only authorized code is executed, and so your GitHub repository
```

Comment:
I don't understand `configure it for openstack mode', the charm is agnostic about how the github runner charm is deployed.

I also think we should avoid using the term "integrate" here, as this could be confused with a juju integration (and the charm does not integrate with the runners via a juju integration, instead it exposes an http server that is called by the github runner charm and the runners, as you described at the bottom of this doc).